package.json

```diff
@@ -39,15 +39,15 @@
     "npm-package-arg": "^6.0.0",
     "rimraf": "^2.6.2",
     "safe-buffer": "^5.1.0",
-    "update-notifier": "^2.3.0",
+    "update-notifier": "^4.1.0",
     "which": "^1.3.0",
     "y18n": "^4.0.0",
     "yargs": "^14.2.3"
   },
   "devDependencies": {
     "cross-env": "^5.1.3",
     "json": "^9.0.6",
-    "marked-man": "^0.2.1",
+    "marked-man": "^0.7.0",
```

Comment:
Also possible that this doesn't need to be updated, as its just a devDependency, and won't be brought in in production